packages/containers/src/ComponentForm/kit/createTriggers.test.js

```diff
@@ -87,6 +89,21 @@ describe('createTriggers', () => {
 			done();
 		});
 	});
+	it('should handle security specified by default', done => {
+		cmf.sagas.http.setDefaultConfig({
+			security: {
+				CSRFTokenCookieKey: 'otherToken',
+				CSRFTokenHeaderKey: 'X-CSRF-OTHER',
+			},
+		});
```

Comment:
you set a default value, you should unset it at the end of the test

Reply:
I've changed it in favor of using a mock. 👍